web/src/data/recoil.ts

```diff
@@ -110,12 +122,41 @@ export const vaporConnectionConfig = selector<ConnectionConfig | undefined>({
   },
 });
 
+export const portalConnectionConfig = selector<ConnectionConfig | undefined>({
+  key: "portalConnectionConfig",
+  get: async ({ get }) => {
+    const portalHostnameValue = get(portalHostname);
+    const portalCredentialsValue = get(portalCredentials);
+
+    if (portalCredentialsValue) {
+      let decodedCredentials;
+      try {
+        decodedCredentials = atob(portalCredentialsValue);
+      } catch (e) {
+        console.error("error conecting to Portal", { e });
```

Comment:
this error message doesn't seem correct, also why are you wrapping the error in an object?

Reply:
it was for debugging, I forgot to remove.